packages/pangea-sdk/pangea/tools_util.py

```diff
@@ -1,26 +1,39 @@
 # Copyright 2022 Pangea Cyber Corporation
 # Author: Pangea Cyber Corporation
 
+import enum
 import io
 import json
 import os
 import sys
 import typing as t
-from datetime import datetime, timezone, date
+from datetime import date, datetime, timezone
 
 from pangea.config import PangeaConfig
 from pangea.services import Audit
 
 
-class Root(t.TypedDict):
+class TestEnvironment(str, enum.Enum):
+    PRODUCTION = "PROD"
```

Comment:
we use dev/stg/lve in the pangea repo, might be worth using the same nomenclature here?

Reply:
@ggallien13 thoughts? I don't have any problem to update all SDKs to that nomenclature, but I should need you to update env var

Reply:
Eh, I prefer PROD, but it's good to be consistent. Let's change it to LVE.